pkg/services/annotations/annotationsimpl/xorm_store.go

```diff
@@ -578,3 +653,8 @@ func (r *xormRepositoryImpl) executeUntilDoneOrCancelled(ctx context.Context, sq
 		}
 	}
 }
+
+type annotationTag struct {
```

Comment:
Backported from later versions; utility for reading tags with xorm, the tests depend on it.